tests/playbooks/tests_ipv6_dns_search.yml

```diff
@@ -0,0 +1,123 @@
+---
+- hosts: all
+  tasks:
+    - name: Configure the static IPv4 and IPv6 is not configured
+      import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: br-example
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+              dns: 8.8.8.8
```

Comment:
Please don't use production IP addresses in tests.

Reply:
I used the address blocks reserved for documentation.